test/libsolidity/SolidityExecutionFramework.cpp

```diff
@@ -28,12 +29,27 @@ using namespace dev;
 using namespace dev::solidity;
 using namespace dev::solidity::test;
 
+ExecutionFramework::boostArg::boostArg()
+{
+	ipcPath = "~/.ethereum/geth.ipc";
```

Comment:
I think we should never attempt to connect to the default path because it could mess up "real" nodes.
Perhaps a better way would be to have `ipcPath` empty in the beginning and call BOOST_FAIL if it is still empty at the end.